src/bot.coffee

```diff
@@ -44,15 +44,18 @@ class SlackBot extends Adapter
     # TODO: check this value when connection finishes (even if its a reconnection)
     # TODO: build a map of enterprise users and local users
     @needsUserListSync = true
-    @client.loadUsers @usersLoaded
+    unless @client.disableUserSync
```

Comment:
does the `@client` object actually have a property called `disableUserSync`? i know its passed in as `options` in the constructor of this object, which means it should be in `@options.disableUserSync`, right?

Reply:
Yeah, you're right, that's a copy-paste error while I was half asleep from reading the project's test files :laughing: it's supposed to be `@options`